### PR TITLE
OK dokument (Independent Publisher)

### PR DESCRIPTION
**Pull request by SoliteaSlovensko**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

okdokument_delete_document
okdokument_download_document
okdokument_signature_request1
okdokument_signature_waiting_trigger
okdokument_upload_action

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hello @SoliteaSlovensko,

Thank you very much and congratulations on your new submission!

Please sign the CLA, this step is necessary. We can't start the review process without it.

After this step is done, our Certification Engineers will review the PR within 1-2 weeks.

Please let us know if you have any questions.

Thank you very much again for working with us.

**Comment by SoliteaSlovensko** [comment]
Hi, thank you. CLA is signed.
I would like to ask if further action is needed from the my site. Because this is my first connector and I am not sure.

In Github is one validation error with description PowerShell exited with code '1'. So I am not sure if I have to take some action and if yes, what should be fixed.

Thank you


Tomáš Szarka | Consultant
+421 915 788 019 | ***@***.******@***.***> | www.solitea.sk<https://eur03.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.solitea.sk%2F&data=04%7C01%7Cpeter.gajdosech%40solitea.sk%7C0a572f87542e47023f9e08d9dc07f3e2%7C6e0a5f8317284956bdf4ce37760cd214%7C0%7C0%7C637782749260980647%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=E3nAapfKjopfHpvrznzPw9zFyGuosxrIKEnqv94Onx4%3D&reserved=0>
Solitea Slovensko, a.s., Pluhova 50/A, 831 03  Bratislava, Slovakia

***@***.***<https://eur03.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.solitea.cz%2F&data=04%7C01%7Cpeter.gajdosech%40solitea.sk%7C0a572f87542e47023f9e08d9dc07f3e2%7C6e0a5f8317284956bdf4ce37760cd214%7C0%7C0%7C637782749260980647%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=SwyV1W2CH01qhAEQxWJfBkMG6nK4tiLUunt93F1zc4s%3D&reserved=0>

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:132]: @SoliteaSlovensko empty schemas are not allowed.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:101]: @SoliteaSlovensko please add value to this property.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:277]: @SoliteaSlovensko empty schemas are not allowed, you can remove it if you don't need this schema. You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) from the documentation.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,681 @@
...
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:299]: @SoliteaSlovensko please add value here.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,681 @@
...
+                  "x-ms-visibility": "internal"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:306]: @SoliteaSlovensko please add value here.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,681 @@
...
+          }
+        ]
+      }
+    },
+    "/document/{Document ID}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:383]: @SoliteaSlovensko empty response schemas are not allowed, please refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,681 @@
...
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:405]: @SoliteaSlovensko please add value here.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,681 @@
...
+                  "x-ms-visibility": "internal"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:412]: @SoliteaSlovensko please add value here.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,681 @@
...
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:430]: @SoliteaSlovensko already mentioned above, empty schemas are not allowed.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,681 @@
...
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:512]: @SoliteaSlovensko please add value here.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,681 @@
...
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:537]: @SoliteaSlovensko empty schema, mentioned above.

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 12]
```diff
@@ -0,0 +1,681 @@
...
+              "x-ms-visibility": "internal"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:608]: @SoliteaSlovensko we can keep empty schemas for trigger operations. For action operations they have to be defined or deleted.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @SoliteaSlovensko! Please check comments after the review.
Let me know if you have any questions, thank you!

File: independent-publisher-connectors/OK dokument/apiDefinition.swagger.json [hunk 13]
```diff
@@ -0,0 +1,681 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OK dokument",
+    "description": "OK dokument delivers an easy to use and customizable electronic signature and digital document distribution platform that integrates seamlessly into your existing systems",
+    "version": "1.0.0",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/OK dokument/apiDefinition.swagger.json:6]: @SoliteaSlovensko please change the version to 1.0

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by v-EgorKozhadei** [comment]
Hello @SoliteaSlovensko, 

I hope you are doing well. 

Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.  

Please let us know if you have any questions. 

Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @SoliteaSlovensko,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by sriyen-msft** [comment]
@SoliteaSlovensko 
Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**